src/Symfony/Contracts/Service/ServiceSubscriberTrait.php

```diff
@@ -22,7 +22,7 @@
 trait ServiceSubscriberTrait
 {
     /** @var ContainerInterface */
-    private $container;
+    private $serviceContainer;
```

Comment:
Hum, looking at the code again, the property is the only way to access the container. Which means this change would break anyone using 100% guaranteed.
That's no-go from BC perspective.
Making the property protected might be the way to go.